includes/Demo_Content.php

```diff
@@ -100,157 +98,92 @@ private function localize_texts( $content ) {
+				esc_html_x( 'Our shapes are quite basic for now but they act as masks. Drag an image or video into the mask.', 'demo content', 'web-stories' ),
 
-		// Page 8.
+			// Page 8.
 
-		$content = str_replace(
-			'L10N_PLACEHOLDER_8_1',
-			/* translators: demo content used in the "Get Started" story */
-			esc_html_x( 'EMBED VISUAL STORIES', 'demo content', 'web-stories' ),
-			$content
-		);
+			'L10N_PLACEHOLDER_8_1' => /* translators: demo content used in the "Get Started" story */
+				esc_html_x( 'EMBED VISUAL STORIES', 'demo content', 'web-stories' ),
 
-		$content = str_replace(
-			'L10N_PLACEHOLDER_8_2',
-			/* translators: demo content used in the "Get Started" story */
-			esc_html_x( 'Embed stories into your blog post. Open the block menu & select the Web Stories block. Insert the story link to embed your story. That\'s it!', 'demo content', 'web-stories' ),
-			$content
-		);
+			'L10N_PLACEHOLDER_8_2' => /* translators: demo content used in the "Get Started" story */
+				esc_html_x( 'Embed stories into your blog post. Open the block menu & select the Web Stories block. Insert the story link to embed your story. That\'s it!', 'demo content', 'web-stories' ),
 
-		// Page 9.
+			// Page 9.
 
-		$content = str_replace(
-			'L10N_PLACEHOLDER_9_1',
-			addslashes(
+			'L10N_PLACEHOLDER_9_1' => addslashes(
 				wp_kses(
-				/* translators: demo content used in the "Get Started" story */
+					/* translators: demo content used in the "Get Started" story */
 					_x( '<span style="font-weight: 100; color: #fff">READ ABOUT </span><span style="font-weight: 600; color: #fff">BEST PRACTICES</span><span style="font-weight: 100; color: #fff"> FOR CREATING A SUCCESSFUL WEB STORY</span>', 'demo content', 'web-stories' ),
 					[
 						'span' => [ 'style' => [] ],
 					]
 				)
 			),
-			$content
-		);
 
-		$content = str_replace(
-			'L10N_PLACEHOLDER_9_2',
-			/* translators: demo content used in the "Get Started" story */
-			esc_html_x( 'https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/', 'demo content', 'web-stories' ),
-			$content
-		);
+			'L10N_PLACEHOLDER_9_2' => /* translators: demo content used in the "Get Started" story */
+				esc_html_x( 'https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/', 'demo content', 'web-stories' ),
 
-		$content = str_replace(
-			'L10N_PLACEHOLDER_9_3',
-			/* translators: demo content used in the "Get Started" story */
-			esc_html_x( 'Best practices for creating a successful Web Story', 'demo content', 'web-stories' ),
-			$content
-		);
+			'L10N_PLACEHOLDER_9_3' => /* translators: demo content used in the "Get Started" story */
+				esc_html_x( 'Best practices for creating a successful Web Story', 'demo content', 'web-stories' ),
+		];
+
+		foreach ( $replacements as $search => $replacement ) {
+			str_replace( $search, $replacement, $content );
```

Comment:
```suggestion
			$content = str_replace( $search, $replacement, $content );
```
Isn't this?

Reply:
Indeed it is!

Fixed in 3962bda12ffc726f518ec5d41b9beecdea099b24